src/commons/BrowserUtils.ts

```diff
@@ -203,6 +205,21 @@ export namespace BrowserUtils {
     );
   }
 
+  /**
+   *
+   * @param selector element selector
+   */
+  export function isEnabled(selector: string): boolean {
```

Comment:
this should be ExpectEnabled returning void
You need to add isEnabled that returns element.isEnabled() without wait for it

Reply:
suggested another solution please review.